From 2988a4ad320b7a4870cfa055bf7afd009424a15f Mon Sep 17 00:00:00 2001
From: Viktor Ashirov <vashirov@redhat.com>
Date: Mon, 28 Jul 2025 13:16:10 +0200
Subject: [PATCH] Issue 6901 - Update changelog trimming logging - fix tests

Description:
Update changelog_trimming_test for the new error message.

Fixes: https://github.com/389ds/389-ds-base/issues/6901

Reviewed by: @progier389, @aadhikar (Thanks!)
---
 .../suites/replication/changelog_trimming_test.py      | 10 +++++-----
 1 file changed, 5 insertions(+), 5 deletions(-)

diff --git a/dirsrvtests/tests/suites/replication/changelog_trimming_test.py b/dirsrvtests/tests/suites/replication/changelog_trimming_test.py
index 2d70d328e..27d19e8fd 100644
--- a/dirsrvtests/tests/suites/replication/changelog_trimming_test.py
+++ b/dirsrvtests/tests/suites/replication/changelog_trimming_test.py
@@ -110,7 +110,7 @@ def test_max_age(topo, setup_max_age):
     do_mods(supplier, 10)
 
     time.sleep(1)  # Trimming should not have occurred
-    if supplier.searchErrorsLog("Trimmed") is True:
+    if supplier.searchErrorsLog("trimmed") is True:
         log.fatal('Trimming event unexpectedly occurred')
         assert False
 
@@ -120,12 +120,12 @@ def test_max_age(topo, setup_max_age):
         cl.set_trim_interval('5')
 
     time.sleep(3)  # Trimming should not have occurred
-    if supplier.searchErrorsLog("Trimmed") is True:
+    if supplier.searchErrorsLog("trimmed") is True:
         log.fatal('Trimming event unexpectedly occurred')
         assert False
 
     time.sleep(3)  # Trimming should have occurred
-    if supplier.searchErrorsLog("Trimmed") is False:
+    if supplier.searchErrorsLog("trimmed") is False:
         log.fatal('Trimming event did not occur')
         assert False
 
@@ -159,7 +159,7 @@ def test_max_entries(topo, setup_max_entries):
     do_mods(supplier, 10)
 
     time.sleep(1)  # Trimming should have occurred
-    if supplier.searchErrorsLog("Trimmed") is True:
+    if supplier.searchErrorsLog("trimmed") is True:
         log.fatal('Trimming event unexpectedly occurred')
         assert False
 
@@ -169,7 +169,7 @@ def test_max_entries(topo, setup_max_entries):
         cl.set_trim_interval('5')
 
     time.sleep(6)  # Trimming should have occurred
-    if supplier.searchErrorsLog("Trimmed") is False:
+    if supplier.searchErrorsLog("trimmed") is False:
         log.fatal('Trimming event did not occur')
         assert False
 
-- 
2.49.0

